schema/deploy/shipping/views.sql

```diff
@@ -2310,7 +2310,11 @@ create or replace view shipping.scan_enrollments_v1 as
         scan_study_arm,
         priority_code,
         puma,
-        neighborhood_district
+        neighborhood_district,
+        case
+            when sample is null then 'false'
+            else 'true'
+        end as kit_received
```

Comment:
I also considered doing 0/1 here. Thoughts on the column value?

Reply:
I'd lean towards a boolean as being the clearest indicator of the potential values here. Does it still evaluate as a boolean if true/false is wrapped in single quotes?

Reply:
Ah, I was not even thinking of producing these in the native PostgreSQL boolean types -- I was only thinking of the downstream use cases. I think you're right -- we should use boolean true/false here. I fixed up this change into my commit.

Reply:
Actually, with that small change, it's clearer to me now that this conditional can be reduced to 
```
        sample is not null as kit_received
```

Reply:
I just re-pushed the commit with this simplification.